packages/core/src/amazonqFeatureDev/session/session.ts

```diff
@@ -209,6 +212,7 @@ export class Session {
     }
 
     public async insertNewFiles(newFilePaths: NewFileInfo[]) {
```

Comment:
If a customer applies files from the file tree, but never clicks "Accept remaining", do we count anything in this approach?

Reply:
Yes. This method is called by both the accept changes button and also by the individual file accept action button.